docs/getting-started.md

```diff
@@ -174,6 +174,35 @@ If the output of this command ends with `CP4NA orchestration tests passed! ✅`
 
 Eventually your access token will expire, resulting in authentication errors when using lmctl. When this occurs, run `login` again to reauthenticate OR check out the [Automatic Reauthentication](#automatic-reauthentication) section below to save your credentials in the lmctl config file so lmctl may automatically re-authenticate you when the token expires.
 
+## Okta Authentication
+
+> For any environment where Okta authentication is used
+
+Run `lmctl login`, changing the values to valid parameters for your environment:
+
+```
+lmctl login $API_GATEWAY --okta --client <Client> --client-secret <secret> -u <user> -p <password> --scope <scope>  --auth-address https://<okta server endpoint> --auth-server-id <auth_server_id>```
```

Comment:
Can we have a follow-up story which gives an example of how to obtain the relevant argument values from Okta? I think we'll have a lot of developers/testers in need of that kind of information

Reply:
Yes we can create another story, I have already attached doc for Okta configuration in the issue.